event model - maggie - UNSURE WHERE TO PUT IM SORRY

```diff
@@ -0,0 +1,72 @@
+import { roles } from '@/types/dataModel/roles';
+import {
+  VolunteerEntity,
+  backgroundCheckStatuses,
+} from '@/types/dataModel/volunteer';
+import { Model, Schema, model, models } from 'mongoose';
+//idk if we need the above but it was from the compassion ministries? so i left it in
+
+const EventModel = new Schema(
+  {
+    ID: {
```

Comment:
We do not need ID in the schema. It will be added when we add data to the database

Reply:
bet